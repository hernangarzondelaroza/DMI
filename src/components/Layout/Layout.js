import React from "react";
import "./Layout.scss";

import { Container } from "react-bootstrap";
import Footer from "./Footer/Footer";
import Header from "./Header/Header";

import PropTypes from "prop-types";

const layout = (props) => {
  return (
    <Container>
      <Header />
      <main>{props.children}</main>
      <Footer />
    </Container>
  );
};

layout.propTypes = {
  children: PropTypes.any
};

export default layout;
