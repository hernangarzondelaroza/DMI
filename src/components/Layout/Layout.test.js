import React from 'react';
import renderer from 'react-test-renderer';
import Layout from './Layout';

describe('Layout', () => {
  it('Should take a snapshot', () => {
    const component = renderer
      .create(<Layout />);
    expect(component).toMatchSnapshot();
  })
})