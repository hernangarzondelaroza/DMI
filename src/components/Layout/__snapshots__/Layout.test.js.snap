// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout Should take a snapshot 1`] = `
<div
  className="container"
>
  <div
    className="row"
  >
    <header
      className="header"
    >
      <h1>
        Technical Test
      </h1>
      <img
        className="logo"
        src="https://v1.nitrocdn.com/LYkBDVHVdldIrognCFlxEKUhZLCymUBO/assets/static/optimized/rev-674e467/wp-content/uploads/2020/11/white-no-sub.svg"
      />
    </header>
  </div>
  <main />
  <div
    className="row"
  >
    <footer
      className="footer fixed-bottom"
    >
      by Hernan Garzon de la Roza
    </footer>
  </div>
</div>
`;
